gcp/deployment-packages/autoscale-byol/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling BYOL solution
+This directory contains CloudGuard IaaS deployment package for Check Point Autoscaling (BYOL) solution published in the [GCP Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/checkpoint.vsec?tab=Overview).
+
+# How to deploy the package manually
+To deploy the Deployment Manager's package manually, without using the GCP Marketplace, follow these instructions:
+1. [Download](https://github.com/CheckPointSW/CloudGuardIaaS/archive/master.zip) the CloudGuardIaaS repository
+2. Browse to the CloudGuardIaaS/gcp/deployment-packages/autoscale-byol/ directory
```

Comment:
this line is no longer needed since in #1 we downloaded the files. now we can go to #3

Reply:
Done